src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -53,9 +75,8 @@ public abstract class NodeModel : ModelBase, IRenderPackageSource<NodeModel>, ID
         private string name;
         private ElementState state;
         private string toolTipText = string.Empty;
```

Comment:
why do we still need the `private string toolTipText` ?

Reply:
Removed.